lib/recipient_interceptor.rb

```diff
@@ -10,8 +10,8 @@ def delivering_email(message)
     add_custom_headers message
     add_subject_prefix message
     message.to = @recipients
-    message.cc = []
-    message.bcc = []
+    message.cc = nil if message.cc
+    message.bcc = nil if message.bcc
```

Comment:
Why `nil` only `if message.cc` and only `if message.ccc`? Should this always be `nil`?

```ruby
message.cc = nil
message.bcc = nil
```

Reply:
The way Mail works is a bit convoluted and I can't give you a straight answer unfortunately but if you set `message.cc = nil` and it's already `nil` it actually gets set to `[]`

It looks something like

```ruby
message.cc        #=> nil
message.cc = nil  #=> nil
message.cc        #=> []
message.cc = nil  #=> nil
message.cc        #=> nil
message.cc = nil  #=> nil
message.cc        #=> []
# repeats
```

Same seems to apply to `bcc`